sphinx-doc/module-md-pair.rst

```diff
@@ -56,10 +53,7 @@ md.pair
         Morse,
         Moliere,
         ReactionField,
-        revcross,
         SLJ,
-        square_density,
         table,
```

Comment:
Can we remove all pair potentials that are not currently updated? I know this is not the purpose of the PR but does need to be done.